proto/cmp/types/v1alpha/traveller.proto

```diff
@@ -47,11 +40,27 @@ enum TravellerType {
   TRAVELLER_TYPE_SENIOR = 4;
 }
 
+message ExtensiveTraveller {
+  // Guest number, the lowest number is the lead-pax. This ID is also used for
+  // referencing services linked to specific participants, like baggage.
+  int32 traveller_id = 1;
+
+  //Gender from the enum below
+  GenderType gender = 2;
+
+  string first_names = 3;
+  string last_name = 4;
+  string second_last_name = 5;
```

Comment:
Why `first_names` is plural and `last_name` is not? And why we have the `second_last_name` and not `second_first_name`? Yes, it sounds funny. :) 

I suggest we have only `first_names` and `last_names` as both can be multiple names.

Reply:
that would cause most of the systems to have to split/merge the multiple lastnames into different fields. Systems that do not have second last name can just use last_name and put multiple values

Reply:
OK, lets change first name and last name as repeated fields and document the explanation as discussed

Reply:
done